<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Archiv</class>
 <widget class="QDialog" name="Archiv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Архив</string>
  </property>
  <widget class="QLabel" name="L_ARCHIV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Архив</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_BOX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_FOLDER">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_RECORD">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_SEARCH">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_BOX">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Box</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_FOLDER">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_RECORD">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_NUMBER">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NUMBER">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="L_OGRN">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ОГРН</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_OGRN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="TV_RESULT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>491</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
